zap/src/main/java/org/parosproxy/paros/view/MainMenuBar.java

```diff
@@ -113,10 +113,11 @@ private void initialize() {
         this.add(getMenuOnline());
         this.add(getMenuHelp());
     }
+
     /**
-     * This method initializes menuEdit
+     * This method initializes (if null) and returns {@code JMenu} menuEdit
```

Comment:
Better just say it's a getter for the edit menu (same for others), that's being lazily initialised is just an implementation detail (something not needed at all since it's being called from the constructor).